src/electron/views/device-connect-view/components/android-setup/prompt-connected-start-testing-step.tsx

```diff
@@ -43,7 +43,7 @@ export const PromptConnectedStartTestingStep = NamedFC<CommonAndroidSetupStepPro
         return (
             <AndroidSetupStepLayout {...layoutProps}>
                 <DeviceDescription {...descriptionProps}></DeviceDescription>
-                <Button
+                <DefaultButton
```

Comment:
Is there any visual impact here?

Reply:
No, it's visually and behaviorally identical